src/clj/rems/db/email_outbox.clj

```diff
@@ -0,0 +1,99 @@
+(ns rems.db.email-outbox
```

Comment:
this could be folded into the rems.email ns if you decide to create it

Reply:
Or make `outbox.cljs` if that is a good direction. I guess we'll have several of these and they'll somewhat replace the poller concept?